packages/react-preferences/src/components/StyledToggle.tsx

```diff
@@ -0,0 +1,113 @@
+import styled from "styled-components";
+
+export const StyledToggle = styled.div`
```

Comment:
do you think we will want to reuse this in other components?  i wonder if we need a common package?  prob ok for now

Reply:
probably not right away, but will move it to common package if that changes